### ISSUE TITLE
Experimental & work-in-progress C++14 multithreaded compile-time entity-component-system library

### ISSUE THREAD

**Issue by bjadamson** [conan-io/wishlist#31]
https://github.com/SuperV1234/ecst

Header only!!

But requires boost hana (also header only!!). Is that easy to model in conan?

I'm using linux clang 3.9 (but it's header only, so it doesn't matter, right??)

Thanks!!

**Comment by memsharded** [comment]
Yes, header only is easy to model in conan, and Boost also has an option, just set ``Boost:header_only=True`` when you install, or in your conanfile. 

Yes, for header only, the settings should be totally removed, no settings in the conanfile. It should be used if you want to develop a ``test_package`` because that has to be built and run, and you can use normal ``settings`` for that.
